tests/backup_restore_test.go

```diff
@@ -368,6 +368,21 @@ func TestPITR(t *testing.T) {
 		time.Sleep(1 * time.Second)
 		targetTs := time.Now().Unix()
 
+		dropStmt := fmt.Sprintf(`DROP DATABASE %s;`, databaseName)
+		_, err = mysqlDB.ExecContext(ctx, dropStmt)
+		a.NoError(err)
+
+		rows, err := mysqlDB.Query(fmt.Sprintf(`SHOW DATABASES LIKE '%s';`, databaseName))
+		a.NoError(err)
+		defer rows.Close()
+		for rows.Next() {
+			var s string
+			err := rows.Scan(&s)
+			a.NoError(err)
+			a.FailNow("Database still exists after dropped")
+		}
+		a.NoError(rows.Err())
```

Comment:
Call `rows.Close()` if `rows` is not used anymore.

Reply:
It's here
https://github.com/bytebase/bytebase/pull/1821/files#diff-678220f1f4dd964016f34084b20da23ceb49cb46dcb6c8e98a401f8dccefa2b0R377

Reply:
Imagine that you would like to add another line of `mysqlDB.query` in the future, and you have to remember to call `rows.Close()` before opening another connection.

Reply:
Change to using different variable names for different query results, which improves readability and also reduces repeated `rows.Close()`.

Reply:
It's not about variable names, it's about the connections that `db.Query` opens. You should always close the connection by calling `rows.Close()` if it won't be used anymore. Otherwise your program might get blocked, if you do something like the following.
```go
row1, _:=db.Query()
defer row1.Close() 
row2, _:=db.Query()
defer row2.Close() 
row3, _:=db.Query()
defer row3.Close() 
row4, _:=db.Query()
defer row4.Close() 
```